File: Deno/examples/PaymentAPI/getOrderByHash.ts
import {QvicklyPaymentAPI, env} from "../../PaymentAPI.ts";

const paymentAPI = new QvicklyPaymentAPI(env["EID"], env["SECRET"]);
const order = await paymentAPI.call("getOrderByHash", {
  hash: "123456abc123456abc123456abc12345",
});
console.log(order);

File: Perl/examples/PaymentAPI/getOrderByHash.pl
#!/usr/bin/perl
use strict;
use warnings;
use JSON::PP;
use Data::Dumper;
use lib '../..';
require "PaymentAPI.pl";
require "LoadEnv.pl";
LoadEnv('../../.env');

my $test = 1;
my $debug = 0;

# Credentials for Auth
my $id = $ENV{"EID"};
my $key = $ENV{"SECRET"};

my $api = PaymentAPI->new($id, $key, $test, $debug);
my $values = {
    "hash" => "123456abc123456abc123456abc12345",
};
print(Dumper($api->call("getOrderByHash", $values)));

1;